_pages/blog.md

```diff
@@ -29,7 +29,7 @@ pagination:
   </div>
   {% endif %}
 
-{% if site.display_tags or site.display_categories %}
+{% if site.display_tags.size > 0 or site.display_categories.size > 0 %}
```

Comment:
What if someone removes `display_tags` and `display_categories` from `_config.yml`? Maybe we should use both checks.

Reply:
Wouldn't it be better to use the old check?
It doesn't make sense to have an empty ```display_tags``` or ```display_categories``` section. If you don't want to display any tags or categories then just remove the entire section

Reply:
I think the same, but I believe adding an empty list check wouldn't hurt.

Reply:
true

Reply:
Hey all,
Thanks for the feedback! 🙌 I’ve updated the conditional check as suggested to handle cases where display_tags and display_categories might be missing in _config.yml. The new condition now ensures both the existence and size of these fields are checked.